--- a/src/behavior_server.cpp	2024-08-24 08:29:00.000000000 +0800
+++ b/src/behavior_server.cpp	2025-05-15 17:38:17.061546908 +0800
@@ -142,7 +142,7 @@
   std::string local_costmap_topic, global_costmap_topic;
   std::string local_footprint_topic, global_footprint_topic;
   std::string robot_base_frame;
-  double transform_tolerance;
+  double transform_tolerance = 0.0f;
   get_parameter("local_costmap_topic", local_costmap_topic);
   get_parameter("global_costmap_topic", global_costmap_topic);
   get_parameter("local_footprint_topic", local_footprint_topic);
