--- a/include/nav_2d_utils/parameters.hpp	2025-05-15 17:52:17.094725348 +0800
+++ b/include/nav_2d_utils/parameters.hpp	2025-05-15 17:52:31.304997324 +0800
@@ -64,7 +64,7 @@
   const nav2_util::LifecycleNode::SharedPtr & nh, const std::string & param_name,
   const param_t & default_value)
 {
-  param_t value;
+  param_t value = 0;
   nav2_util::declare_parameter_if_not_declared(
     nh, param_name,
     rclcpp::ParameterValue(default_value));
